libsolidity/parsing/Token.cpp

```diff
@@ -155,16 +155,15 @@ tuple<Token::Value, unsigned int, unsigned int> Token::fromIdentifierOrKeyword(s
 			) {
 				int n = parseSize(positionX + 1, _literal.end());
 				if (
-					0 < m && m < 256 &&
-					0 < n && n < 256 &&
+					m + n > 0 &&
```

Comment:
yes. Parse size can return -1 because that's the error statement. If you look in the function though, it is set to lexical cast to an unsigned int...and if there is a -1, there's no way they can make is equal to 0 on a modulo 8 condition...so I think we're fairly safe.